cmd/ike/cmd/develop.go

```diff
@@ -31,12 +31,15 @@ func NewDevelopCmd() *cobra.Command {
 		},
 		RunE: func(cmd *cobra.Command, args []string) error { //nolint[:unparam]
 
-			sessionClose, err := sessions(cmd)
+			sessionState, sessionClose, err := sessions(cmd)
 			if err != nil {
 				return err
 			}
 			defer sessionClose()
 
+			// HACK: need contract with TP cmd?
+			cmd.Flags().Set("deployment", sessionState.DeploymentName)
```

Comment:
Another possible `HACK` could be to create a shared variable (this is a single process anyway) and mutate that one instead of relying on the internals of Cobra. So something like `TelepresenceOptions` where we store what to pass and (maybe) how to map it to the CLI call. WDYT?

Reply:
Ok this sucks even more